### DISCUSSION TITLE
How to create Content Control elements

### DISCUSSION THREAD

**Discussion by knopmickael** [dolanmiu/docx#2639] [Q&A] [answered]
I was looking for the **Content Control** component, but did not find it on the docs. Can someone help on how to use it or even if it already has been implemented? Thanks

![image](https://github.com/dolanmiu/docx/assets/49726886/a66bdd9e-03ba-4d31-a56c-a01ca72e6523)

**Comment by anti-the-social** [comment, marked as answer]
Hi!

To my knowledge, it isn't a part of the library. At least yet. 

I would've like to see it too tho.

Probably someone brave should do PR :) 
I didn't have time for that unfortunately
